src/content/pages/for-nerds.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Credits
```

Comment:
```suggestion
title: Under the hood
```

Reply:
Or just "How it's built"?

Also, I feel like 'for-nerds' is a weird URL. Instead, I'd go with `under-the-hood` or `how-its-made` or `technical-details` or even just `for-the-curious`.